mangaki/mangaki/models.py

```diff
@@ -355,7 +379,8 @@ def __str__(self):
 
 
 class Tag(models.Model):
-    title = models.TextField(unique = True)
+    title = models.CharField(max_length=255)
+    anidb_tag_id = models.IntegerField(null=True, unique=True)
```

Comment:
I think we shouldn't set `null=True` and like we said, drop all older tags.
Or, to do the right thing, we should copy the old tags into another model and perform a migration of old tags to new tags with AIDs.